COMMUNITY.md

```diff
@@ -492,6 +492,36 @@ In all cases, compatibility pieces should be marked with compiler/preprocessor
 annotations to print warnings at compile time, pointing to the appropriate
 update path.  A `-Werror` build should fail if compatibility bits are used.
 
+### Release Process/Schedule
+
+FRR employs a <MAJOR>.<MINOR>.<BUGFIX> versioning scheme.
+
+* MAJOR - Significant new features or multiple minor features
+  A example of a MAJOR feature is a New Routing Protocol
+* MINOR - Smaller Features
+  A example of a MINOR feature is the addition of the BGP Shutdown feature.
+* BUGFIX - Fixes for actual bugs and/or security issues.
+
+We will pull a new development branch for the next release every 4 months.
+The current schedule is Feb/June/October 1.  The decision for a MAJOR/MINOR
+release is made at the time of branch pull based on what has been received
+the previous 4 months.  The branch name will be dev/MAJOR.MINOR.  At
+this point in time the master branch configure.ac and packaging systems
+will be updated to reflect the next possible release name to allow
+for easy distinguishing.  Additionally the new dev branch will have
+these files updated too.
+
```

Comment:
The expectation / process covering what happens during the 1st month should be defined / captured. (Is it more than changing configure.ac and packaging? if not, why a month, if yes, what...)